### PR TITLE
sdl_image: add version 2.8.2, fix compilation error for webp and avif

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **sdl_image/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by toge** [comment]
There is a version conflict.
Waiting for https://github.com/conan-io/conan-center-index/pull/21772

**Comment by PerikiyoXD** [comment]
TL;DR: current status to build 2.8.2:
- https://github.com/conan-io/conan-center-index/pull/21681#issuecomment-1891016321 shows `libXvMC` is not installed.
- d7b80d60329bd8ee6185c15d77ebd9748bc4fafa removed `libXvMC` from [xorg](https://github.com/conan-io/conan-center-index/tree/d48a7d283c0b002b337f941eaf191e2aff637009/recipes/xorg)
  - It is supposed that RHEL9 does not use `libXvMC`, but was removed for every platform, or so it seems.
